tests/test_surface/conftest.py

```diff
@@ -0,0 +1,12 @@
+import numpy as np
+import pytest
+
+
+@pytest.fixture
+def default_surface():
```

Comment:
This is how I would suggest removing the test surface, that way it is contained in the tests and does not leak into production code.